flight/Libraries/math/lqg.c

```diff
@@ -0,0 +1,505 @@
+#include "lqg.h"
```

Comment:
need our copyright notice.   Also, is there any source (e.g. sympy, matlab) that this came from that should be checked in too for license compliance?

Reply:
The MATLAB stuff was done just in some scratch file, with matrixes of syms and then run the formula and paste the output into a C file. I can redo these for inclusion, since they were trivial.

Will add the copyright notices.